test/e2e/multi-cluster-deployment/setup-kubestellar.sh

```diff
@@ -13,27 +13,27 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+set -x # echo so users can understand what is happening
 set -e # exit on error
 
 echo "Create a Kind hosting cluster with nginx ingress controller and KubeFlex operator"
 echo "-------------------------------------------------------------------------"
-kflex init --create-kind &> /dev/null
+kflex init --create-kind
 echo "Kubeflex kind cluster created."
 
 echo "Create an inventory & mailbox space of type vcluster running OCM (Open Cluster Management) directly in KubeFlex. Note that -p ocm runs a post-create hook on the vcluster control plane which installs OCM on it."
 echo "-------------------------------------------------------------------------"
-kflex create imbs1 --type vcluster -p ocm &> /dev/null
+kflex create imbs1 --type vcluster -p ocm
```

Comment:
Because of the way this test runs in CICD and the output is captured, this kflex create and other like it generate plenty of useless messages that overwhelm the test output.  Perhaps we should add a parameter to the test case that by default will display the output and in CICD will pipe to /dev/null or to a file?